### PR TITLE
Upgrade to Django 1.10.1

### PR DESCRIPTION
**Pull request by glogiotatidis**

### CODE REVIEW

**Comment by glogiotatidis** [comment]
This is currently blocked by django-session-csrf being incompatible with Django 1.10. There's an open pull request to fix that: https://github.com/mozilla/django-session-csrf/pull/30

@peterbe maybe you can push things forward?

**Comment by peterbe** [comment]
I left some comments https://github.com/mozilla/django-session-csrf/pull/30 but once that lands I can take care up updating pypi. Then we can proceed on this PR.

**Comment by glogiotatidis** [comment]
@peterbe updated!

File: {{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/urls.py [hunk 1]
```diff
@@ -1,14 +1,16 @@
 from django.conf import settings
 from django.conf.urls import include, url
 from django.contrib import admin
+from django.views.static import serve as static_serve
```
Comment (peterbe) [inline {{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/urls.py]: Why did it need to be aliased? What's wrong with just `serve`?

**Comment by peterbe** [comment]
We might need to go back to django-session-csrf and work on this. But I just created a fresh new install based on this branch and it won't even start. 
screen shot 2016-10-03 at 8 49 47 am

The fault lies with django-session-csrf but we have to take responsibility of that.

**Comment by peterbe** [comment]
Let's hope @moggers87 finds the time to complete the python 3 support. 
https://github.com/mozilla/django-session-csrf/issues/20#issuecomment-162044638

I just can't accept that we say that sugardough is compatible with django 1.10 but doesn't work in Python 3.

File: {{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/urls.py (hunk 1 above)
Comment (glogiotatidis) [inline {{ cookiecutter.project_name }}/{{ cookiecutter.project_name }}/urls.py]: I just find `static_serve` more descriptive

**Comment by glogiotatidis** [comment]
Fair enough, let's wait that for a week. @moggers87 is already working on py3

**Comment by peterbe** [comment]
I tested your PR together with the python3 branch that @moggers87 is working on and now my recently created sugardough python 3.5 project worked. 
See https://github.com/mozilla/django-session-csrf/pull/36#pullrequestreview-2727721

The only testing I did was to load the home page. But that didn't work before. 

So let's pause on this PR until we can also include a new django-session-csrf version hashed in.

**Comment by glogiotatidis** [comment]
woohoo! @peterbe updated, feel free to squash merge

**Pull request merged**